lib/calendar.js

```diff
@@ -58,15 +61,27 @@ export default class Calendar {
         csreq=0
       WHERE client='?' AND service='kTCCServiceCalendar'`.replace(/\n/g, ' ');
     } else {
-      query = this.xcodeVersion.major < 10 ?
-        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0);` :
-        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0, 0, 0, 0, 0);`;
+      let count = this.columnCount;
+      if (_.isEmpty(this.columnCount)) {
+        try {
+          const {stdout} = await execSQLiteQuery(db, `pragma table_info('access')`);
+          // stdout has a number of records, each starting with `cid`
+          count = this.columnCount = stdout.trim().split('cid').filter((record) => !_.isEmpty(record)).length;
```

Comment:
I think it might be easier to read this line if formatted like

```
stdout.trim()
  .split
  .filter
...
```

Reply:
Sure.